tests/cross-compile/get-dimensions-stride.slang

```diff
@@ -0,0 +1,58 @@
+//TEST(compute):COMPARE_COMPUTE_EX(filecheck-buffer=CHECK):-vk -compute -shaderobj -emit-spirv-directly
```

Comment:
I think you just need one -vk test without `-emit-spirv-directly`, and it will be run twice with both spirv and glsl backends.